docs/Refund.md

```diff
@@ -6,28 +6,64 @@ If you are looking for information about requesting a refund from a merchant, pl
 
 **Refunds** are one of the applications built on top of the [Pull Payments](./PullPayments.md) feature.
 
-Up until now, merchants had to manually issue refunds.
-The process required a lot of back and forth between a sender and a receiver. The merchant had to ask for a refund address, calculate the preferred rate, manually add that to the wallet and broadcast the transaction.
-
-A merchant just needs to go to the invoice, click **Issue refund** , select the appropriate rate and share a link with a customer.
-
-The customer will then input their address and claim a refund. Once a customer claims a refund, the merchant will get a notification and can authorize it in a click.
-Future versions of BTCPay Server will enable instant payouts as an option, without the need for merchant approval.
+On this page, we will talk you through the flow of creating Refunds. 
+There are a short few steps to create the Refund for the customer. 
+After the refund has been issued, we will continue to the [Payouts](./Payouts.md) page to talk about the refund processing. 
 
 ## Create a refund
 
-1. After an invoice has been confirmed, go in the Invoices page and click `Details` on the invoice.
+1. After an invoice has been confirmed but needs a refund, go in the Invoices page and click `Details` on the invoice.
```

Comment:
A few invoice statues can be refunded (partially paid, settled, etc, so confirmed isn't correct at least in terms of our invoice statuses.
```suggestion
1. To refund an invoice,  go in the `Invoices` page and click `Details` on the invoice.
```